### PR TITLE
fix: Remove tabindex

### PR DESCRIPTION
**Pull request by makandre**
Closes IBM/carbon-components-angular#2184

~Change role from `search` to `searchbox`~

~I believe `searchbox` is the correct widget role based on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#2._widget_roles~

Remove the unnecessary `tabindex`

#### Changelog

**New**

* {{new thing}}

**Changed**

* [src/search/search.component.html](https://github.com/IBM/carbon-components-angular/compare/master...makandre:carbon-components-angular:patch-2#diff-d18ddec221752adaa51aabfcf9d75769a9608dc390c37b3722eb87cb9433cbb6)

**Removed**

* {{removed thing}}

### CODE REVIEW

**Review by Dagdelo — DISMISSED** [review]

File: src/search/search.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		'bx--toolbar-search-container-expandable': tableSearch && expandable,
 		'bx--toolbar-search-container-active': tableSearch && expandable && active
 	}"
-	role="search"
+	role="searchbox"
```
Comment (Akshat55) [inline src/search/search.component.html:17]: Seems like the accessibility error is caused by `tabindex`, not the role.

```suggestion
	role="search"
```

image

File: src/search/search.component.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
...
 	[attr.aria-label]="ariaLabel"
 	tabindex="0"
```
Comment (Akshat55) [inline src/search/search.component.html:19]: `div` should not have tabindex. 

```suggestion
```

Comment (Akshat55) [reply src/search/search.component.html:19]: We don't need tabindex here since `input` element will have it.

**Review by Akshat55 — CHANGES_REQUESTED** [review]
Thanks for taking on this issue, @makandre! 

I've requested a few changes.

Also, for the new commit, please include `fix:` at the beginning!

**Review by Akshat55 — APPROVED** [review]

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**